doc/content/getting-started/api/_index.md

```diff
@@ -0,0 +1,179 @@
+
+```bash
+$ curl --location \
+     --header 'Authorization: Bearer NNSXS.XXXXXXXXX' --header 'Content-Type: application/json' \
+     --request POST 'https://thethings.example.com/api/v3/applications/app1/devices' \
+     --data-raw '{
+      "end_device": {
+        "ids": {
+          "device_id": "newdev1",
+          "dev_eui": "0000000000000000",
+          "join_eui": "0000000000000000"
+        },
+        "join_server_address": "thethings.example.com",
+        "network_server_address": "thethings.example.com",
+        "application_server_address": "thethings.example.com"
+      },
+      "field_mask": {
+        "paths": [
+          "join_server_address",
+          "network_server_address",
+          "application_server_address",
+          "ids.dev_eui",
+          "ids.join_eui"
+        ]
+      }
+    }'
+```
+
+Register MAC settings in the Network Server:
+
+```bash
+$ curl --location \
+     --header 'Authorization: Bearer NNSXS.XXXXXXXXX' --header 'Content-Type: application/json' \
+     --request PUT 'https://thethings.example.com/api/v3/ns/applications/app1/devices/newdev1' \
+     --data-raw '{
+      "end_device": {
+        "supports_join": true,
+        "lorawan_version": "1.0.2",
+        "ids": {
+          "device_id": "newdev1",
+          "dev_eui": "0000000000000000",
+          "join_eui": "0000000000000000"
+        },
+        "supports_class_c": false,
+        "lorawan_phy_version": "1.0.2-b",
+        "frequency_plan_id": "EU_863_870_TTN"
+      },
+      "field_mask": {
+        "paths": [
+          "supports_join",
+          "lorawan_version",
+          "ids.device_id",
+          "ids.dev_eui",
+          "ids.join_eui",
+          "supports_class_c",
+          "lorawan_phy_version",
+          "frequency_plan_id"
+        ]
+      }
+    }'
+```
+
+Register the `DevEUI` and `JoinEUI` in the Application Server:
+
+```bash
+$ curl --location \
+     --header 'Authorization: Bearer NNSXS.XXXXXXXXX' --header 'Content-Type: application/json' \
+     --request PUT 'https://thethings.example.com/api/v3/as/applications/app1/devices/newdev1' \
+     --data-raw '{
+      "end_device": {
+        "ids": {
+          "device_id": "newdev1",
+          "dev_eui": "0000000000000000",
+          "join_eui": "0000000000000000"
+        }
+      },
+      "field_mask": {
+        "paths": [
+          "ids.device_id",
+          "ids.dev_eui",
+          "ids.join_eui"
+        ]
+      }
+    }'
+```
+
+Finally, register the `DevEUI`, `JoinEUI`, cluster addresses, and keys in the Join Server:
+
+```bash
+$ curl --location \
+     --header 'Authorization: Bearer NNSXS.XXXXXXXXX' --header 'Content-Type: application/json' \
+     --request PUT 'https://thethings.example.com/api/v3/js/applications/app1/devices/newdev1' \
+     --data-raw '{
+      "end_device": {
+        "ids": {
+          "device_id": "newdev1",
+          "dev_eui": "0000000000000000",
+          "join_eui": "0000000000000000"
```

Comment:
For `key` you use `X`s, here you use `0`s, I'd prefer a consistent format for context-specific data, perhaps just use `X` to fool-proof this, such that people don't start copy-pasting commands and creating a bunch of 0000000000000000 DevEUI and JoinEUI devices?